docs/source/llms/tracing/index.rst

```diff
@@ -480,6 +490,104 @@ for model/API invocations to the active MLflow Experiment.
             :width: 100%
             :align: center
 
+    .. tab:: Ollama
+
+        .. raw:: html
+
+            <h3>Ollama Automatic Tracing</h3>
+
+        |
+
+        `Ollama <https://github.com/ollama/ollama>`_ is an open-source platform that enables users to run large language models (LLMs) locally on their devices, such as Llama 3.2, Gemma 2, Mistral, Code Llama, and more.
```

Comment:
Question: Are we ok with adding tracing to the caller side only for this integration? Wonder if users also want to instrument the server side (Ollama framework side).

Reply:
No. Tho I haven't read server side code of ollama and how customizable it is by users. Have you seen any use cases?

Reply:
Got it, just wondering if people who use tracing for the usage of Ollama may also care about what's happening inside the Ollama side. Ollama is written in go and c, so it's not straightforward to add tracing to the library from mlflow.

Reply:
We don't know the answer - and I agree it's not easy to add internal trace.